src/product-card-rating.scss

```diff
@@ -0,0 +1,19 @@
+.product-card__rating {
+  display: flex;
+  margin: 0 $productCardMargin 24px $productCardMargin;
```

Comment:
again, you need just bottom margins. but give `padding: 16px`(or smth, I'm not sure) for  card